### PR TITLE
initial commit of docs, including structure and autodoc of specific m…

### PR DESCRIPTION
**Pull request by MasonEgger**
…odules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ If you want to quickly get started synthesizing data with **Gretel.ai**, simply
 
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/gretelai/trainer/blob/main/notebooks/trainer-examples.ipynb)
 
-## Join our Slack Workspace
+## Join they Synthetic Data Community Discord
```
Comment (csbailey5t) [inline README.md:25]:
```suggestion
## Join the Synthetic Data Community Discord
```

**Review by zredlined — APPROVED** [review]

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+* Multiple `model types supported <https://docs.gretel.ai/synthetics/models>`_\:
+    * `Gretel-LSTM` model type supports text, tabular, time-series, and conditional data generation.
+    * `Gretel-CTGAN` model type supports tabular and conditional data generation.
+    * `Gretel-GPT` natural language synthesis based on an open-source implementation of GPT-3 (coming soon).
+    * `Gretel-DGAN` multi-variate time series based on DoppelGANger (coming soon).
+
+Train Synthetic Data in as Little as Three Lines of Code!
+---------------------------------------------------------
+
+#. Install the Gretel CLI and Gretel Trainer either on your system or in your Notebook
```
Comment (csbailey5t) [inline docs/index.rst:28]:
```suggestion
#. Install the Gretel CLI and Gretel Trainer either on your system or in your Notebook.
```

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+#. Add your `Gretel API <https://console.gretel.cloud>`_ key via the Gretel CLI.
+
+   Use the Gretel client to store your API key to disk. This step is optional, the trainer will prompt for an API key in the next step.
+
+   .. code-block:: bash
+
+      gretel configure
+
+#. Train or fine-tune a model using the Gretel API
```
Comment (csbailey5t) [inline docs/index.rst:46]:
```suggestion
#. Train or fine-tune a model using the Gretel API.
```

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+.. image:: https://colab.research.google.com/assets/colab-badge.svg
+   :target: https://colab.research.google.com/github/gretelai/trainer/blob/main/notebooks/trainer-examples.ipynb
+   :alt: Open in Colab
+
+Join the Synthetic Data Community Discord
+-----------------------------------------
+
+If you want to be part of the Synthetic Data Community to receive announcements of the latest releases,
+ask questions, suggest new features or participate in the development meetings, please join
```
Comment (csbailey5t) [inline docs/index.rst:76]:
```suggestion
ask questions, suggest new features, or participate in the development meetings, please join
```

File: docs/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
+Quickstart
+==========
+
+Initial Setup
+-------------
+
+#. Install the Gretel CLI and Gretel Trainer either on your system or in your Notebook
```
Comment (csbailey5t) [inline docs/quickstart.rst:7]:
```suggestion
#. Install the Gretel CLI and Gretel Trainer either on your system or in your Notebook.
```

File: docs/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+        gretel configure
+
+Train Synthetic Data
+--------------------
+.. image:: https://colab.research.google.com/assets/colab-badge.svg
+    :target: https://colab.research.google.com/github/gretelai/trainer/blob/main/notebooks/trainer-examples.ipynb
+    :alt: Open in Colab
+
+#. Train or fine-tune a model using the Gretel API
```
Comment (csbailey5t) [inline docs/quickstart.rst:31]:
```suggestion
#. Train or fine-tune a model using the Gretel API.
```

File: docs/quickstart.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+        df = model.generate()
+
+Conditional Data Generation
+---------------------------
+.. image:: https://colab.research.google.com/assets/colab-badge.svg
+    :target: https://colab.research.google.com/github/gretelai/trainer/blob/main/notebooks/simple-conditional-generation.ipynb
+    :alt: Open in Colab
+
+#. Load and preview the dataset, set seed fields
```
Comment (csbailey5t) [inline docs/quickstart.rst:54]:
```suggestion
#. Load and preview the dataset, and set seed fields.
```

File: docs/quickstart.rst [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+        import pandas as pd
+        from gretel_trainer import trainer
+
+        DATASET_PATH = 'https://gretel-public-website.s3.amazonaws.com/datasets/mitre-synthea-health.csv'
+        SEED_FIELDS = ["RACE", "ETHNICITY", "GENDER"]
+
+        print("\nPreviewing real world dataset\n")
+        pd.read_csv(DATASET_PATH)
+
+#. Train the model
```
Comment (csbailey5t) [inline docs/quickstart.rst:68]:
```suggestion
#. Train the model.
```

File: docs/quickstart.rst [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+    .. code-block:: python3
+
+        # Train model
+        model = trainer.Trainer()
+        model.train(DATASET_PATH, seed_fields=SEED_FIELDS)
+
+#. Conditionally generate data
```
Comment (csbailey5t) [inline docs/quickstart.rst:76]:
```suggestion
#. Conditionally generate data.
```

**Review by csbailey5t — CHANGES_REQUESTED** [review]
Minor punctuation changes for consistency, then good to go

**Review by csbailey5t — APPROVED** [review]

**Pull request merged**